sale_stock_picking_validation_blocking/tests/test_sale_stock.py

```diff
@@ -31,11 +31,9 @@ def setUp(self):
     def test_sale_stock_picking_validation_blocked(self):
         self.sale_order.action_confirm()
         picking = self.sale_order.picking_ids
-        picking.move_lines.write({"quantity_done": 1})
+        picking.move_ids.write({"quantity": 1})
         self.sale_order.action_block_picking_validation()
-        self.assertFalse(picking.show_validate)
```

Comment:
Do you want to check the value of `hide_button_picking_validation_blocked` instead, here and below?